tests/e2e/support/commands.js

```diff
@@ -240,6 +267,39 @@ Cypress.Commands.add('playPointsOpponent', (card) => {
     });
 });
 
+/**
+ * @param card {suit: number, rank: number}
+ */
+Cypress.Commands.add('playPointsSpectator', (card, pNum) => {
```

Comment:
It also goes hand in hand with switching your session data. It might make sense to make this a higher order command that both switches your session data and executes the points move